news/2021-05-12-new-featured-artist-agressor-bunx.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "New Featured Artist: Agressor Bunx"
+date: 2021-05-12 09:00:00 +0000
+---
+
+**Agressor Bunx** joins the lineup as our newest Featured Artist!
+
+![](https://assets.ppy.sh/artists/164/header.jpg)
+
+A trailblazer in electronic scenes for a decade, **Agressor Bunx** is a Ukrainian duo with an insanely fierce sound. Developing their powerful sound over the years from a simple drum & bass feel to deeper neurofunk tones, the two talents have perfected their craft and are still managing to one-up themselves with each new song.
```

Comment:
word variety

edit: eph just committed some rewording but i still think some of this could work, feel free to pick and choose

```suggestion
A trailblazer in electronic scenes for a decade, **Agressor Bunx** is a Ukrainian duo with an insanely fierce sound. Developing their powerful style over the years from a simple drum & bass feel to deeper neurofunk tones, the two talents have perfected their craft and are still managing to one-up themselves with each new song.
```

Reply:
applied most, leaving this one though